aten/src/ATen/core/ivalue.cpp

```diff
@@ -574,11 +574,9 @@ static std::ostream& printMaybeAnnotatedDict(
 static std::ostream& printComplex(std::ostream & out, const IValue & v) {
   c10::complex<double> d = v.toComplexDouble();
   IValue real(d.real()), imag(std::abs(d.imag()));
-  auto sign = "";
+  auto sign = '-';
```

Comment:
one of the few times this actually might be cleaner as a single line ternary?

Reply:
Fixed